driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -343,9 +343,9 @@ public VersionNumber getCassandraVersion() {
    * {@code null}, and the caller should always be aware of this possibility.
    *
    * @return the DSE version the host is running.
-   * @deprecated Please use the <a href="https://github.com/datastax/java-driver-dse">Java Driver
-   *     for DSE</a> if you are connecting to a DataStax Enterprise (DSE) cluster. This method might
-   *     not function properly with future versions of DSE.
+   * @deprecated Please use the <a href="https://github.com/apache/cassandra-java-driver-dse">Java
+   *     Driver for DSE</a> if you are connecting to a DataStax Enterprise (DSE) cluster. This
+   *     method might not function properly with future versions of DSE.
```

Comment:
*lol* Fun additional fact; this comment (and all comments like it) need to be removed in their entirety.  Presumably this reflects logic from before the OSS and DSE drivers were unified.  Since that unification java-driver-dse is no longer maintained and the recommendation actually would be to use the base Java driver.

If we wind up de-unifying things this would change but for now that would be the recommendation.